services/scabbard/libscabbard/src/migrations/diesel/postgres/migrations/2022-06-16-131600-consensus-add-decision-ack/up.sql

```diff
@@ -0,0 +1,22 @@
+-- Copyright 2018-2022 Cargill Incorporated
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+-- -----------------------------------------------------------------------------
+
+CREATE TYPE decision_ack_value AS ENUM ('TRUE', 'FALSE');
+
+ALTER TABLE consensus_2pc_context_participant
```

Comment:
PostgreSQL has a boolean type, which we should use when storing booleans.

Reply:
Fixed